scripts/extensions/auto-tagging-widget/src/auto-tagging.tsx

```diff
@@ -47,13 +49,17 @@ interface IIMatricsFields {
     };
 }
 
-type IEditableData = {original: IAutoTaggingResponse; changes: IAutoTaggingResponse};
+type IEditableData = {
+    original: IAutoTaggingResponse; 
+    changes: IAutoTaggingResponse;
+};
 
 interface IState {
     runAutomaticallyPreference: boolean | 'loading';
     data: 'not-initialized' | 'loading' | IEditableData;
     newItem: INewItem | null;
     vocabularyLabels: Map<string, string> | null;
+    showImageWidget: boolean;
```

Comment:
This should be removed if it's not actually used. It's set to `true` in the constructor and can never be changed. In that case, simply render the component unconditionally.

Reply:
This has been refactored and renamed to "showImagesPreference", and uses the Superdesk preference setting similar to run automatically.